src/transformers/modeling_utils.py

```diff
@@ -2321,6 +2334,31 @@ def from_pretrained(
 
         # Load config if we don't provide a configuration
         if not isinstance(config, PretrainedConfig):
+            # In case users load models without auto mapping
+            if is_peft_available() and peft_adapter_model_id is None:
+                raw_adapter_config_dict_path = cls._check_and_return_if_adapter_model(
+                    pretrained_model_name_or_path,
+                    revision=revision,
+                    use_auth_token=use_auth_token,
+                )
+
+                if raw_adapter_config_dict_path is not None:
+                    raw_adapter_config_dict = json.load(open(raw_adapter_config_dict_path, "r"))
+
+                    if "base_model_name_or_path" in raw_adapter_config_dict:
+                        has_adapter_file = True
+                        peft_adapter_model_id = pretrained_model_name_or_path
+
+                        logger.info(
+                            f"Found adapter file at {peft_adapter_model_id}. Automatically loading adapter model using the base model from {raw_adapter_config_dict['base_model_name_or_path']}"
+                        )
```

Comment:
Needs to be split on several lines.

Reply:
Thanks for noticing, fixed in cd22c543ef06b336dc5735a9fdc149c87c20d9f3

Reply:
Fixed in  ef26bd5